gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
`-all` _should_ be unneeded for kotlin-dsl, see https://github.com/gradle/kotlin-dsl/issues/604

Reply:
Whoops...I remember you mentioned that, but totally forgot. Thanks.